README.md

```diff
@@ -1,7 +1,7 @@
 ![CI](https://github.com/JuliaDiff/ForwardDiff.jl/workflows/CI/badge.svg) [![Coverage Status](https://coveralls.io/repos/JuliaDiff/ForwardDiff.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/JuliaDiff/ForwardDiff.jl?branch=master)
```

Comment:
While here, the CI badge doesn't link to CI, just to the image. I think this should be:
```suggestion
[![CI](https://github.com/JuliaDiff/ForwardDiff.jl/workflows/CI/badge.svg)](https://github.com/JuliaDiff/ForwardDiff.jl/actions/workflows/ci.yml)
[![Coverage Status](https://coveralls.io/repos/JuliaDiff/ForwardDiff.jl/badge.svg?branch=master&service=github)](https://coveralls.io/github/JuliaDiff/ForwardDiff.jl?branch=master)
```

Reply:
Ah, I didn't notice that. Thanks!